### DISCUSSION TITLE
Invalid JSON data in request body.

### DISCUSSION THREAD

**Discussion by andyweb** [fattureincloud/api#401] [Q&A] [answered]
Ciao, ho questo codice:

`{
  "data": {
    "type": "invoice",
    "entity": {
      "id": 1,
      "name": "Andrea Spurio",
      "vat_number": "47803200154",
      "tax_code": "RSSMRA91M20B967Q",
      "address_street": "Via Italia, 66",
      "address_postal_code": "20900",
      "address_city": "Milano",
      "address_province": "MI",
      "address_extra": "",
      "country": "Italia"
    },
    "date": "2024-01-25",
    "number": 1,
    "numeration": "/fatt",
    "subject": "Scadero",
    "visible_subject": "Scadero",
    "currency": {
      "id": "EUR",
      "exchange_rate": "1.00000",
      "symbol": "€"
    },
    "language": {
      "code": "it",
      "name": "Italiano"
    },
    "items_list": [
            {
                "product_id": 31830339,
                "code": "",
                "name": "Abbonamento Scadero",
                "net_price": 99,
                "category": "software",
                "discount": 0,
                "qty": 1,
                "vat": {
                    "id": 0 #22%
                }
            }
        ]
  }`

perchè mi dice Invalid JSON data in request body?

**Comment by valmoz** [comment]
Buongiorno @andyweb 
Ha questo errore perché il body non è un JSON valido
![image](https://github.com/fattureincloud/api/assets/7254293/2a9ed5f4-2b5d-4b23-b4d7-f288365c17be)

Grazie
Mauro

**Comment by andyweb** [comment]
Grazie ma non trovo l'errore...

Il giorno gio 25 gen 2024 alle ore 16:31 Mauro Valota <
***@***.***> ha scritto:

> Buongiorno @andyweb <https://github.com/andyweb>
> Ha questo errore perché il body non è un JSON valido
> image.png (view on web)
> <https://github.com/fattureincloud/api/assets/7254293/2a9ed5f4-2b5d-4b23-b4d7-f288365c17be>
>
> Grazie
> Mauro
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fattureincloud/api/discussions/401#discussioncomment-8246492>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5NHANXHZA4ZRXJD6SWUCTYQJ3E7AVCNFSM6AAAAABCKWWM4OVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DENBWGQ4TE>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

WIM di Andrea Spurio
62026 San Ginesio (MC)
mobile: 329 8530293
www.italiawim.it

**Reply by valmoz** [reply]
Mancava una parentesi

**Comment by andyweb** [comment]
```json
{
  "data": {
    "type": "invoice",
    "entity": {
      "id": 1,
      "name": "Andrea Spurio",
      "vat_number": "47803200154",
      "tax_code": "RSSMRA91M20B967Q",
      "address_street": "Via Italia, 66",
      "address_postal_code": "20900",
      "address_city": "Milano",
      "address_province": "MI",
      "address_extra": "",
      "country": "Italia"
    },
    "date": "2024-01-25",
    "number": 1,
    "numeration": "/fatt",
    "subject": "Scadero",
    "visible_subject": "Scadero",
    "currency": {
      "id": "EUR",
      "exchange_rate": "1.00000",
      "symbol": "€"
    },
    "language": {
      "code": "it",
      "name": "Italiano"
    },
    "items_list": [
    
    {
                "vat": {
                    "id": 0
                },
                "name": "Scadero",
                "net_price": 99,
                "qty": 1
            }
        ]
  }
}
```

risposta: Il totale dei pagamenti non corrisponde al totale da pagare.

**Reply by valmoz** [reply, marked as answer]
@andyweb può leggere qui: [FAQ](https://developers.fattureincloud.it/docs/FAQs#why-am-i-receiving-a-document-totals-error)

**Reply by valmoz** [reply]
@andyweb è riuscito a risolvere?

**Comment by andyweb** [comment]
si, grazie.

Il giorno mer 31 gen 2024 alle ore 11:51 Mauro Valota <
***@***.***> ha scritto:

> @andyweb <https://github.com/andyweb> è riuscito a risolvere?
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/fattureincloud/api/discussions/401#discussioncomment-8317927>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/AA5NHAKP75ZQ3MXRSFBTPX3YRIO3PAVCNFSM6AAAAABCKWWM4OVHI2DSMVQWIX3LMV43SRDJONRXK43TNFXW4Q3PNVWWK3TUHM4DGMJXHEZDO>
> .
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 

WIM di Andrea Spurio
62026 San Ginesio (MC)
mobile: 329 8530293
www.italiawim.it
